include/random_walks/random_walks.hpp

```diff
@@ -20,6 +20,7 @@
 #include "random_walks/uniform_john_walk.hpp"
 #include "random_walks/uniform_vaidya_walk.hpp"
 #include "random_walks/uniform_accelerated_billiard_walk.hpp"
+#include "random_walks/exponential_hamiltonian_monte_carlo_walk.hpp"
```

Comment:
Please keep header names consistent. If the struct name has exponential after hmc, it should be better if it's the same here.

Reply:
I renamed it accordingly to the previous comment.